### PR TITLE
fix: extra invalidation on image response

### PR DESCRIPTION
**Pull request by rebelchris**
We already invalidate the cloudinary flow, but cloudinary makes it so that the URL will remain the same on a update.
To ensure a browser reload we added a optional version number which is current date timestamp.

WT-1670 #done 

Further discussion: https://dailydotdev.slack.com/archives/C02E2C3C13R/p1692365364463109

### CODE REVIEW

File: src/schema/posts.ts [hunk 1]
```diff
@@ -1410,7 +1410,7 @@ export const resolvers: IResolvers<any, Context> = {
             UploadPreset.PostBannerImage,
             { invalidate: true },
           );
-          updated.image = coverImageUrl;
+          updated.image = `${coverImageUrl}v=${Date.now()}`;
```
Comment (sshanzel) [inline src/schema/posts.ts:1413]: I think we are missing the question mark for the query parameters.

**Review by sshanzel — APPROVED** [review]
Looks great! 🚀

**Comment by idoshamun** [comment]
@rebelchris feels a bit hacky, doesn't cloudinary provide a way to do it via their sdk? Probably there's an argument that returns a new url

**Comment by rebelchris** [comment]
@idoshamun Can look into that :)
As it was already pretty much edge-casey didn't put too much time into it.

**Comment by rebelchris** [comment]
@idoshamun / @sshanzel Went with a force_version if we want to invalidate as well.
It forces the cloudinary datestamp version string instead

**Review by idoshamun — APPROVED** [review]

**Pull request merged**